config_template.py

```diff
@@ -55,6 +57,18 @@
 # This works when "Record a separate audio file of each participant" is enabled.
 INCLUDE_PARTICIPANT_AUDIO = True
 
+# Recording file name format to use when saving files. Reorder or comment out any file name format pieces below to control the file naming pattern.
+# Example: 2023-12-25t143021z__name-of-the-meeting__audio_transcript__ff625374.VTT
+FILE_NAME_FORMAT = [
+     R"RECORDING_START_DATETIME",   # Recording start datetime
+     R"RECORDING_NAME",             # Recording name
+     R"RECORDING_TYPE",             # Recoding type
+     R"FILE_ID",                    # Recording unique file ID
+]
```

Comment:
This is a bit problematic
I use the file_id in the end to detect already downloaded file. I don't think it's a good idea to let users remove it...